schemas/context/experienceevent.example.1.json

```diff
@@ -85,7 +85,31 @@
       "schema:longitude": 139.73237
     }
   },
-  "xdm:marketing": {
-    "xdm:trackingCode": "marketingcampaign111"
+  "xdm:referral": {
+    "xdm:trackingCode": "KKQWX",
```

Comment:
Moving the `xdm:trackingCode` to the proposed  `xdm:referral` suggests that the current page/email/mobile state is not able to set the trackingCode.

Reply:
Interesting point. I would see the `trackingCode` as something that the referring experience sets, opposed to properties like `currentCampaign`, `selectedOffer`, `relevantExperience`, etc. that can be resolved in the current context. 

We can keep it in `xdm:marketing` if it makes more sense.

Reply:
I've moved it back in 1a49af8 @cdegroot-adobe